great_expectations/metrics/multi_column_values/multi_column_values.py

```diff
@@ -0,0 +1,15 @@
+from typing import Literal, Optional
```

Comment:
Can you rename this directory and file `multi_column` to follow the existing pattern

Reply:
Also the `__init__.py` in this directory should import `from .multi_column import MultiColumnMetric`